src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -122,11 +127,14 @@ private void leaseAndProcessZipFile(
 
         leaseAcquirer.ifAcquiredOrElse(
             blobClient,
-            leaseId -> processZipFile(container, blobClient, zipFilename, leaseId),
+            leaseId -> {
+                if (ocrValidationRetryManager.canProcess(blobClient)) {
+                    processZipFile(container, blobClient, zipFilename, leaseId);
+                }
```

Comment:
it may be worth logging something if it's not ready yet imho

Reply:
It is logged in `OcrValidationRetryManager`